app/schemas.py

```diff
@@ -26,10 +26,10 @@ def get_and_validate_json_from_request(request, schema):
             },
             "required": ["subject", "content"]
         },
-        "dvla_org_id": {"type": "string"},
+        "dvla_org_id": {"type": ["string", "null"]},
         "filename": {"type": ["string", "null"]},
         "date": {"type": ["string", "null"]},
     },
-    "required": ["letter_contact_block", "template", "values", "dvla_org_id"],
+    "required": ["letter_contact_block", "template", "values"],
```

Comment:
If filename is always passed in now, should we make it required?

Reply:
👍 Yes, it makes sense to make `filename` required here instead of in a later PR so that it's not possible to have no value for `filename` and no value for `dvla_org_id` either - I've pushed a commit.